Chandra/cmd_states/cmd_states.py

```diff
@@ -382,9 +382,14 @@ def get_state0(date=None, db=None, date_margin=10, datepar='datestop'):
      time.  The conservative default value of ``date_margin=10`` allows for
      processing downtime.  The table is accessed via the ``db`` object.
 
+     If ``date_margin`` is None then no date margin is applied and the most
+     recent available state that meets the date selection criteria (including
+     that pcad_mode == 'NPNT') will be used.  In this case a ``date`` must be
```

Comment:
Otherwise you end up with `date = None` which will fail the subsequent `fetchone` call that expects a correctly formatted ISO date string.  Our common idiom that a `None` date => NOW does not apply in this function call.